### ISSUE TITLE
BrowserTestBaseDefaultThemeRule false positives needs to account for upgrade test

### ISSUE THREAD

**Issue by jibran** [mglaman/phpstan-drupal#234]
Same as #198 but for upgrade path tests.

This rule should also not trigger for subclasses of `UpdatePathTestBase` 

```
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/src/Functional/Update/DerRevUpdateTest.php                                                                                                         
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 
  16     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See https://www.drupal.org/node/3083055, which includes recommendations on which theme to use.  
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/src/Functional/Update/DerUpdate8202Test.php                                                                                                        
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 
  13     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See https://www.drupal.org/node/3083055, which includes recommendations on which theme to use.  
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/src/Functional/Update/DerUpdateTest.php                                                                                                            
 ------ --------------------------------------------------------------------------------------------------------------------------------------------------------- 
  16     Drupal\Tests\BrowserTestBase::$defaultTheme is required. See https://www.drupal.org/node/3083055, which includes recommendations on which theme to use.  
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------------------
